.github/workflows/release-on-tag.yml

```diff
@@ -0,0 +1,30 @@
+name: Tag on merge to master
```

Comment:
```suggestion
name: Release on tag on master
```

This might be clearer? It would be similar to the current filename: `release-on-tag.yml`.

Reply:
Whoops, copy-paste error.